### ISSUE TITLE
Exception in thread "Thread-1" java.lang.RuntimeException: Could not process model 'cluster'.Please make sure that your schema is correct!

### ISSUE THREAD

**Issue by subtleseeker** [swagger-api/swagger-codegen#10212]
##### Description
Says there is an error in schema, despite checking with https://editor.swagger.io/.
Error log:
```
23:41:21.977 [main] DEBUG io.swagger.codegen.v3.cli.SwaggerCodegen - there are not options for command 'langs'
23:41:21.981 [main] DEBUG io.swagger.codegen.v3.cli.SwaggerCodegen - there are not options for command 'version'
23:41:22.050 [Thread-1] DEBUG io.swagger.v3.parser.OpenAPIV3Parser - Loaded raw data: swagger: '2.0'
info:
  title: test api
  description: test api
...
      cluster:
        $ref: '#/definitions/cluster'

23:41:22.065 [Thread-1] DEBUG io.swagger.v3.parser.OpenAPIV3Parser - Parsed rootNode: {"swagger":"2.0","info":{"title":"test api","description":"test api","version":"3.0.0"},"schemes":["http"],"securityDefinitions":{"basicAuth":{"type":"basic"}},"security":[{"basicAuth":[]}],"basePath":"/v3","produces":["application/json"],"parameters":{"uuid":{"name":"uuid","in":"path","type":"string","format":"UUID","required":true,"pattern":"^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"}},"paths":{"/cluster/{uuid}":{"get":{"consumes":["application/json"],"produces":["application/json"],"tags":["cluster"],"summary":"Get cluster details by uuid.","parameters":[{"$ref":"#/parameters/uuid"}],"responses":{"200":{"description":"Success","schema":{"$ref":"#/definitions/cluster_response"}}}}}},"definitions":{"cluster_uuid":{"type":"string","format":"UUID","pattern":"^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"},"cluster":{"type":"object","title":"cluster Definition","properties":{"cluster_uuid":{"$ref":"#/definitions/cluster_uuid"}}},"cluster_response":{"type":"object","title":"cluster Response","properties":{"api_version":{"type":"string"},"cluster":{"$ref":"#/definitions/cluster"}}}}}
23:41:22.216 [Thread-1] DEBUG io.swagger.codegen.v3.config.CodegenConfigurator - getClientOptInput - parsed inputSpecURL /home/nikhil-mundra/tmp/swagger.yaml
Exception in thread "Thread-1" java.lang.RuntimeException: Could not process model 'cluster'.Please make sure that your schema is correct!
        at io.swagger.codegen.v3.DefaultGenerator.generateModels(DefaultGenerator.java:386)
        at io.swagger.codegen.v3.DefaultGenerator.generate(DefaultGenerator.java:779)
        at io.swagger.codegen.v3.cli.cmd.Generate.run(Generate.java:360)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: Pattern must follow the Perl /pattern/modifiers convention. ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$ is not valid.
        at io.swagger.codegen.v3.generators.python.PythonClientCodegen.postProcessPattern(PythonClientCodegen.java:300)
        at io.swagger.codegen.v3.generators.python.PythonClientCodegen.postProcessModelProperty(PythonClientCodegen.java:286)
        at io.swagger.codegen.v3.generators.DefaultCodegenConfig.fromModel(DefaultCodegenConfig.java:1410)
        at io.swagger.codegen.v3.DefaultGenerator.processModels(DefaultGenerator.java:1011)
        at io.swagger.codegen.v3.DefaultGenerator.generateModels(DefaultGenerator.java:375)
        ... 3 more
```

##### Swagger-codegen version
[v3.0.19](https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.19/swagger-codegen-cli-3.0.19.jar)

##### Swagger declaration file content or url
```yaml
swagger: '2.0'
info:
  title: test api
  description: test api
  version: 3.0.0
schemes:
  - http
securityDefinitions:
  basicAuth:
    type: basic
security:
  - basicAuth: []
basePath: /v3
produces:
  - application/json
parameters:
  uuid:
    name: uuid
    in: path
    type: string
    format: UUID
    required: true
    pattern: ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
paths:
  "/cluster/{uuid}":
    get:
      consumes:
        - application/json
      produces:
        - application/json
      tags: ["cluster"]
      summary: Get cluster details by uuid.
      parameters:
        - $ref: '#/parameters/uuid'
      responses:
        200:
          description: Success
          schema:
            $ref: '#/definitions/cluster_response'
definitions:
  cluster_uuid:
    type: string
    format: UUID
    pattern: ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
  cluster:
    type: object
    title: cluster Definition
    properties:
      cluster_uuid:
        $ref: '#/definitions/cluster_uuid'
  cluster_response:
    type: object
    title: cluster Response
    properties:
      api_version:
        type: string
      cluster:
        $ref: '#/definitions/cluster'
```

##### Command line used for generation
```java -jar ~/tmp/swagger-codegen-cli-3.0.19.jar generate -i ~/tmp/swagger.yaml -l python -o ~/tmp/swag```

##### Steps to reproduce
With the given yaml, try generating python-client with the above command will produce this error.

**Comment by PaddySe** [comment]
The exact same exception is thrown when using the csharp generator.

**Comment by subtleseeker** [comment]
Seems it is fixed with version [3.0.26](https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.26/).
Closing the issue.
